src/modules/timepix3/test/Timepix3PixelEventHandlerTest.cpp

```diff
@@ -6,6 +6,7 @@
 
 #include "common/kafka/EV44Serializer.h"
 #include "common/kafka/Producer.h"
+#include "geometry/Config.h"
 #include "geometry/Timepix3Geometry.h"
 #include "gtest/gtest.h"
```

Comment:
#include <>

and specifically

#include <timepix3/geometry/Config.h>
#include <timepix3/geometry/Timepix3Geometry.h>